src/Discord.Net.Interactions/Builders/Modals/Inputs/IInputComponentBuilder.cs

```diff
@@ -0,0 +1,105 @@
+using System;
+using System.Collections.Generic;
+
+namespace Discord.Interactions.Builders
+{
+    /// <summary>
+    ///     Represent a builder for creating <see cref="InputComponentInfo"/>.
+    /// </summary>
+    public interface IInputComponentBuilder
+    {
+        /// <summary>
+        ///     Gets the parent modal of this input component.
+        /// </summary>
+        public ModalBuilder Modal { get; }
```

Comment:
Should interface properties be public?

Reply:
I made them private to be consistent with [commands](https://github.com/Discord-Net-Labs/Discord.Net-Labs/blob/a94f9851c958b35955b692ece52385ccc00aa8ed/src/Discord.Net.Interactions/Builders/Commands/ICommandBuilder.cs).